src/libcore/num/i128.rs

```diff
@@ -14,4 +14,4 @@
 
 #![unstable(feature = "i128", issue="35118")]
 
-int_module! { i128 }
+int_module! { i128, #[unstable(feature = "i128", issue="35118")] }
```

Comment:
The feature name was defined to be `i128_type` here
https://github.com/rust-lang/rust/blob/master/src/libsyntax/feature_gate.rs#L323
Should this match that definition?